Formula/cayley.rb

```diff
@@ -1,89 +1,16 @@
 class Cayley < Formula
-  desc "Graph database inspired by Freebase and Knowledge Graph"
-  homepage "https://github.com/cayleygraph/cayley"
-  url "https://github.com/cayleygraph/cayley.git",
-    :tag      => "v0.7.7",
-    :revision => "dcf764fef381f19ee49fad186b4e00024709f148"
+  desc "An open-source graph database"
+  homepage "An open-source graph database"
+  url "https://github.com/cayleygraph/cayley/releases/download/v0.7.7/cayley_0.7.7_darwin_amd64.tar.gz"
```

Comment:
Don't use pre-built binaries

Reply:
Why not?

Reply:
https://docs.brew.sh/Acceptable-Formulae#we-dont-like-binary-formulae

Reply:
Thanks, will look into it.